### DISCUSSION TITLE
接入openai大模型后，提示词能难调好，翻译效果不尽人意，大家是否有更好的建议

### DISCUSSION THREAD

**Discussion by yxw007** [yxw007/vscode-translate-next#8] [Q&A]
![open_ai_usage](https://github.com/user-attachments/assets/a93c52f7-7864-45d2-8614-4ade0bba2de8)

**Comment by damon98xp** [comment]
I can't see the picture, what's your prompt for now?

![image](https://github.com/user-attachments/assets/bd1d06b0-f4c8-4aa5-9c25-1f9663cc7b89)

**Reply by yxw007** [reply]
> I can't see the picture, what's your prompt for now?
> 
> ! [image](https://private-user-images.githubusercontent.com/1480581/402415376-bd1d06b0-f4c8-4aa5-9c25-1f9663cc7b89.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY3NjQwNzcsIm5iZiI6MTczNjc2Mzc3NywicGF0aCI6Ii8xNDgwNTgxLzQwMjQxNTM3Ni1iZDFkMDZiMC1mNGM4LTRhYTUtOWMyNS0xZjk2NjNjYzdiODkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDExMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAxMTNUMTAyMjU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGYyY2M2Zjk0M2ZmNWI5ZTM1Zjc0MmY4MDZiNWI2ZTc5MDZjM2Y2MzU2YjMwM2E1MmM2MTc2NDcwMTU3NWI5MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.rhCdGMgwkpG1ONXqSoL1H7D3tBOig36U7pQIvnMNZuY)

Sorry, the picture has been updated and can now be displayed normally
